mobile/android/logcat-helper.ts

```diff
@@ -72,6 +72,24 @@ export class LogcatHelper implements Mobile.ILogcatHelper {
 		return logcatStream;
 	}
 
+	public async dump(deviceIdentifier: string): Promise<void> {
+		const adb: Mobile.IDeviceAndroidDebugBridge = this.$injector.resolve(DeviceAndroidDebugBridge, { identifier: deviceIdentifier });
+		const logcatDumpStream = await adb.executeCommand(["logcat", "-d"], { returnChildProcess: true });
```

Comment:
does this command work on all API levels?

Reply:
Tested the command manually on a device running Android 4.2.1(API Level 17) and it works as expected.